core-bundle/src/Resources/config/controller.yml

```diff
@@ -59,6 +59,8 @@ services:
 
     Contao\CoreBundle\Controller\ContentElement\HtmlController: ~
 
+    Contao\CoreBundle\Controller\ContentElement\UnfilteredHtmlController: ~
```

Comment:
This should be placed after `ContentElement\TemplateController`. I wonder why the service linter did not stumble across this. 🤔